src/analysis/exponential.lean

```diff
@@ -1152,4 +1156,26 @@ by simp only [rpow_def, (complex.of_real_pow _ _).symm, complex.cpow_nat_cast,
 by simp only [rpow_def, (complex.of_real_fpow _ _).symm, complex.cpow_int_cast,
   complex.of_real_int_cast, complex.of_real_re]
 
+lemma rpow_le_rpow (x y z: ℝ) (h : 0 ≤ x) (h₁ : x ≤ y) (h₂ : 0 ≤ z): x^z ≤ y^z :=
+begin

```

Comment:
I think the best proof of this is to first prove `1 ≤ x` and `0 ≤ z` implies `1 ≤ x ^ z`. This should be straightforward if you prove something about `log_nonneg`. Then just use the fact that `y ^ z` = `x ^ z * (y / x) ^ z`. That way you don't have quite as many ifs to split.